WorkloadManagementSystem/Utilities/test/Test_PilotCStoJSONSynchronizer.py

```diff
@@ -115,10 +139,30 @@ def tearDown(self):
 
 class Test_PilotCStoJSONSynchronizer_sync(PilotCStoJSONSynchronizerTestCase):
 
+  @patch('DIRAC.WorkloadManagementSystem.Utilities.PilotCStoJSONSynchronizer.requests', new=Mock())
   def test_success(self):
     synchroniser = PilotCStoJSONSynchronizer()
+    synchroniser.pilotFileServer = 'value'
+    res = synchroniser._syncJSONFile()
+    assert res['OK']
```

Comment:
```suggestion
    assert res['OK'], res['Message']
```
would print `res['Message']` in case `res['OK']` is `False`

Reply:
and add it also for the other `assert` statements

Reply:
Adder the message part to the assert statements